### ISSUE TITLE
The Sunday Times

### ISSUE THREAD

**Issue by lcthomas** [whatevery1says/DownloadAgent#3]
# # retrieve source id
source_results = session.search_sources(query='The Sunday Times')
print(source_results)

Doesn't return any results, even though "The Sunday Times" is listed as a source in LexisNexis and returns results using Nexis Uni.

**Comment by scottkleinman** [comment]
The Sunday Times and The Times (issue #1) were not in the download of (supposedly) all sources @jeremydouglass gave to me on Friday. Just in case that's of some help in bug hunting.

**Comment by lcthomas** [comment]
Yes -- we've discovered that not all of the sources are included in that download of (supposedly) all sources. The New York Times isn't in there, for example, and we can definitely get that through the API. I'm not sure that these cases are bugs: they aren't bad id's, for example. We may just not be able to get these sources from the API, unfortunately. Will keep investigating this issue when I can.
